src/bidiMapper/domains/events/EventManager.ts

```diff
@@ -275,24 +285,13 @@ export class EventManager {
     }
     return result.sort((e1, e2) => e1.id - e2.id);
   }
-}
 
-const EVENT_NAMES = new Set([
-  // keep-sorted start
-  ...Object.values(ChromiumBidi.BiDiModule),
-  ...Object.values(ChromiumBidi.BrowsingContext.EventNames),
-  ...Object.values(ChromiumBidi.Log.EventNames),
-  ...Object.values(ChromiumBidi.Network.EventNames),
-  ...Object.values(ChromiumBidi.Script.EventNames),
-  // keep-sorted end
-]);
-
-function checkEventName(name: string) {
-  if (
-    !EVENT_NAMES.has(name as never) &&
-    !name.startsWith('cdp.') &&
-    name !== 'cdp'
-  ) {
-    throw new InvalidArgumentException(`Unknown event: ${name}`);
+  static isSupportedEvent(name: string) {
```

Comment:
The function name is a bit misleading: it sounds like it would return a boolean.

How about: `assertSupportedEvent` or `ensureSupportedEvent`? (I prefer the former)